### PR TITLE
Closes AgileVentures/MetPlus_tracker#194

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: features/agency_admin.feature [hunk 1]
```diff
@@ -45,11 +45,11 @@ Scenario: toggle data tables
   Then "123 Main Street" should be visible
   And I should see "Smith, John"
   Then I click the "Hide People" link
-  And I wait 2 seconds
+  And I wait 1 seconds
   Then "Smith, John" should not be visible
-  Then I click the "Show People" link
-  And I wait 1 second
-  Then "Smith, John" should be visible
+  # Then I click the "Show People" link
```
Comment (joaopapereira) [inline features/agency_admin.feature]: This is no longer valid?

**Comment by joaopapereira** [comment]
Please review my comment, but otherwise looks good

File: features/agency_admin.feature (hunk 1 above)
Comment (patmbolger) [inline features/agency_admin.feature]: It is valid but this particular test (line 52) seems to fail on CI 2 times out of 3.  When I have time I can cycle back and try to figure out why, but for mow it just gets in everyone's way with having to run CI multiple times.

**Comment by joaopapereira** [comment]
I am ok with your answer. going to merge this pr

**Pull request merged**